src/Tizen.NUI/src/public/DragAndDrop/DragEvent.cs

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright(c) 2022 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using System.ComponentModel;
+using Tizen.NUI.Binding;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Drag event type.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum DragType
+    {
+      Enter,
+      Leave,
+      Move,
+      Drop
+    }
+
+    /// <summary>
+    /// This specifies drag event.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public struct DragEvent
+    {
+        public DragType Type;
+        public Vector2 Position;
+        public string Data;
+    }
+}
```

Comment:
this is trivial thing but could you add new line at the end of file?
reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline